documentation/src/mkdocs.yml

```diff
@@ -21,7 +21,27 @@ site_description: 'Documentation to help you get familiar with Pravega'
 repo_url: 'https://github.com/pravega/pravega'
 edit_uri: 'tree/master/documentation/src/docs'
 
-theme: 'material'
+# theme: 'readthedocs'
```

Comment:
I'm not sure why we have all these lines below commented out. Should we simply remove them?

Reply:
just a hold over from testing -- will clean it up